shared/gh/js/utils/gh.utils.time.js

```diff
@@ -247,7 +247,7 @@ define(['exports', 'gh.constants', 'moment', 'moment-timezone'], function(export
         });
 
         // Return the date of the first lecture day
-        return moment(term.start).add({'days': 2, 'hours': 1}).toISOString();
+        return moment(term.start).add({'days': 2, 'hours': -((new Date()).getTimezoneOffset() / 60)}).toISOString();
```

Comment:
The thing is that we don't know if the timezone is going to be +8 or -8 hours. When the tz is -8 hours the local time needs to be `date + 8` and the math suggests that -(-8) will be +8 and -(+8) will be -8.